validator/sawtooth_validator/journal/journal.py

```diff
@@ -234,6 +237,9 @@ def get_current_root(self):
     def get_block_store(self):
         return self._block_store
 
+    def add_permission_verifier(self, permission_verifier):
+        self._permission_verifier = permission_verifier
```

Comment:
This feels a little off given that the `_permission_verify` can be `None`for a period of time, from the Journal's perspective.  Clearly, this is due to the circular dependencies between the two.  It requires ordering of calls out side of the journal.  Can the PermissionVerifier be initialized with the current state root in another way?  I.e. not via the Journal.

Minimally, the `_init_subprocesses` function should through an error if it is `None`, before creating the subprocesses, to ensure that the journal isn't created without on.